test/unit/server-ha-standby-service.bats

```diff
@@ -168,3 +168,63 @@ load _helpers
       yq -r '.spec.ports | map(select(.port==8200)) | .[] .name' | tee /dev/stderr)
   [ "${actual}" = "https" ]
 }
+
+@test "server/ha-standby-Service: vault externalTrafficPolicy set to Local lowercase" {
+  cd `chart_dir`
+
+  local actual=$(helm template \
+      --show-only templates/server-ha-standby-service.yaml \
+      --set 'server.service.externalTrafficPolicy=local' \
+      --set 'server.ha.enabled=true' \
+      . | tee /dev/stderr |
+      yq -r '.spec.externalTrafficPolicy | tee /dev/stderr)
```

Comment:
This is missing a `'` and so are the other tests you added here, which is causing failures when running the tests.

Reply:
Just committed the change from my phone :)